src/run.js

```diff
@@ -0,0 +1,116 @@
+const cozyClient = require('cozy-konnector-libs/dist/libs/cozyclient')
+const errors = require('cozy-konnector-libs/dist/helpers/errors')
+const { log } = require('cozy-konnector-libs')
+
+const {
+  fetchAndSaveTrips,
+  fetchAndSaveManualEntries,
+  fetchTripsMetadata
+} = require('./lib')
+const { getFirstAndLastTripTimestamp } = require('./trace-requests')
+const {
+  canSaveNextTripsChunk,
+  restartKonnector,
+  createChunks
+} = require('./utils')
+const { saveAccountData } = require('./save')
+const { TRIPS_CHUNK_SIZE } = require('./const')
+const client = cozyClient.new
+
+const getTimeout = () => {
+  const maxExecutionTimeSeconds = parseInt(process.env.COZY_TIME_LIMIT, 10)
+  return maxExecutionTimeSeconds - 100
```

Comment:
Does it mean we remove 100seconds to the COZY_TIME_LIMIT? Can you explain why this particular value?

Reply:
Because I like round values :)
in cozy-konnector-libs, we set an arbitrary timeout value to  COZY_TIME_LIMIT - 60s. I increased to 100 to be more cozy in time.